snuba/consumers/consumer.py

```diff
@@ -204,15 +298,17 @@ def join(self, timeout: Optional[float] = None) -> None:
 
 
 class ProcessedMessageBatchWriter(
-    ProcessingStep[Union[None, JSONRowInsertBatch, ReplacementBatch]]
+    ProcessingStep[Union[None, JSONRowInsertBatch, ReplacementBatch, ValuesInsertBatch]]
 ):
     def __init__(
         self,
         insert_batch_writer: InsertBatchWriter,
```

Comment:
I guess this should also be made optional, since the insert with values doesn't need to set this up